src/app.rs

```diff
@@ -477,6 +486,10 @@ where
         info!("Version: {}", env!("CARGO_PKG_VERSION"));
         info!("Platform: {}", env!("VERGEN_TARGET_TRIPLE"));
         info!("Amethyst git commit: {}", env!("VERGEN_SHA"));
+        if let Some(sentry) = option_env!("SENTRY_DSN") {
+            info!("Sentry DSN: {}", sentry);
```

Comment:
and probably dont want to print this atleast not at an info level?

Reply:
No, it is fine for the DSN to be public. The service has rate limiting and IP restriction built in if we need it.